lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
I don't think you can get busy or serialization errors inside a transaction, only once you try to commit.  So, it would re-try the entire transaction in that case.  Or did I misunderstand the example?